cmd/tailscale/cli/switch.go

```diff
@@ -67,11 +79,19 @@ func switchProfile(ctx context.Context, args []string) error {
 	}
 	var profID ipn.ProfileID
 	for _, p := range all {
-		if p.Name == args[0] {
+		if p.ID == ipn.ProfileID(args[0]) {
 			profID = p.ID
 			break
 		}
 	}
+	if profID == "" {
```

Comment:
So I guess we're accepting names or IDs - not a bad idea because it doesn't break existing workflows. Maybe add a little comment about why we do this